internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -205,7 +205,7 @@ func (r *Reconciler) reconcileMonitoringSecret(
 	intent.Data = make(map[string][]byte)
 
 	if len(existing.Data["password"]) == 0 || len(existing.Data["verifier"]) == 0 {
-		password, err := util.GeneratePassword(util.DefaultGeneratedPasswordLength)
+		password, err := util.GenerateASCIIPassword(32)
```

Comment:
I'd prefer we increment the `util.DefaultGeneratedPasswordLength` constant vs. hard coding it to 32 everywhere.

Reply:
The named constant is used for both spec users and our internal/system users. We know what passwords are tolerable for the tools of the internal users. A single constant for all of them doesn't seem right, but we can address that later.

I'll revert this.